### PR TITLE
docs: Uninstall Gateway

### PR DESCRIPTION
**Pull request by jamilbk**
https://discord.com/channels/1228082899023298741/1239053156965679115

### CODE REVIEW

File: website/src/app/kb/administer/uninstall/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import SupportOptions from "@/components/SupportOptions";
+import { TabsGroup, TabsItem } from "@/components/DocsTabs";
+import Alert from "@/components/DocsAlert";
+
+# Uninstall Firezone Gateway
+
+This guide covers how to uninstall the Firezone Gateway from your server(s).
+
+{/* add Alert and link to client uninstall docs when that's complete */}
```
Comment (jamilbk) [inline website/src/app/kb/administer/uninstall/readme.mdx]:
```suggestion
{/* TODO: add Alert and link to client uninstall docs when that's complete */}
```

File: website/src/app/kb/administer/uninstall/readme.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+<TabsGroup>
+<TabsItem title="Docker" active>
+
+1. Stop the container:
+
+   ```
+   docker stop firezone-gateway
+   ```
+
+1. Remove the container its volumes:
```
Comment (bmanifold) [inline website/src/app/kb/administer/uninstall/readme.mdx]:
```suggestion
1. Remove the container and its volumes:
```

File: website/src/app/kb/administer/uninstall/readme.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   ```
+
+</TabsItem>
+</TabsGroup>
+
+## Step 2 (optional): Delete the data from the admin portal
+
+If you want to remove the Gateway's data from the admin portal as well, navigate
+to the Site where the Gateway was running, click `(see all)` next
+`Online Gateways`, and then delete the offline Gateway corresponding to the
```
Comment (bmanifold) [inline website/src/app/kb/administer/uninstall/readme.mdx]:
```suggestion
to the Site where the Gateway was running, click `(see all)` next to the
`Online Gateways` header, and then delete the offline Gateway corresponding to the
```

**Review by bmanifold — APPROVED** [review]
Couple of tiny grammar things, but other than that LGTM

**Pull request merged**